src/test/java/com/wikia/webdriver/elements/oasis/pages/TemplateEditPage.java

```diff
@@ -34,7 +36,13 @@ public TemplateClassification getTemplateClassification() {
   }
 
   public boolean isEditAreaDisplayed() {
-    return editArea.isDisplayed();
+    try {
+      wait.forElementVisible(editArea);
+      return editArea.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)